mods/d2k/rules/starport.yaml

```diff
@@ -21,7 +21,6 @@ trike.starport:
 	Inherits: trike
 	Buildable:
 		Queue: Starport
-		Prerequisites: starport
```

Comment:
It is redundant. Compare with the other entries - if they're purchasable in the Starport queue, which is only activated once you actually build the Starport, there is no need to put starport in the prerequisites.